tests/proxy.rs

```diff
@@ -161,6 +163,19 @@ async fn test_quit_lifecycle() {
         .expect("app exits without error");
 }
 
+#[tokio::test]
+async fn test_readiness() {
```

Comment:
we already do this check in `TestApp.ready()` which is called in all other tests. If we do want a test _just_ for readiness we can do that but should re-use the code (which also has a backoff so we don't send a ton of requests in a hot loop)

Reply:
I completely missed the whole `test_helpers` directory! 🤦🏾 I'll modify those tests to use the correct port and remove this

Reply:
FYI - it looks like `ready()` doesn't check the status code because the changes in this PR should've made admin requests result in a 404